usagov/vote-gov-drupal
# Bug/vote 1477 untranslated links

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-1477](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1477&assignee=611272e0eb7aef00694b8a97)

## Description (optional)
This PR will add conditional logic for state pages for untranslated links.  The idea is that if a translation does not have a link in that language and is left blank in the CMS then the sting translation will concat (in English) for it so the user will know how ever when there is a link in the overide field that is translated it will remove the (in English) string

## Deployment and testing (required, if applicable)
### Testing steps
1. Pull down branch
2. start local 
3. run `lando retune`
4. run `lando drush uli` and log in as a super user 
5. visit the site and navigate to  `Alabama` state page
6. click through each translation and you should see (in English) at the end of each link 
7. choose a few languages (Spanish and Talalog are the best examples) and click the edit button. 
8. add `http://example.com` to each override field
9. return back to the page and verify that the (in English) string is removed

>**Note**
> I was not able to check the other templates with updating the links I received this error message 
>  "Non-translatable fields can only be changed with updating the original language 
> I was able to check that the logic was working before updating the string translations and there were double instances of (in English) not sure why `Alamaba` let me know it but not others
